magnum/solvers/search_oracle.py

```diff
@@ -0,0 +1,78 @@
+from itertools import cycle
+
+import stl
+import numpy as np
+
+import traces
+from functools import reduce
+
+def random_series(g):
+    times = (g.model.dt*t for t in g.times)
+    return traces.TimeSeries(zip(times, np.random.random(len(g.times))))
```

Comment:
Shouldn't this depend on the radius and u?

Reply:
No  I have taken care of that when I assign 'u'

Reply:
ah I see that now